File: Feitos em Java/2803.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
    public static void main(String[] args) throws IOException {
        Scanner entrada = new Scanner (System.in);
    	String regiao = entrada.nextLine();
        if("para".equals(regiao)||"roraima".equals(regiao)||"acre".equals(regiao)||"amapa".equals(regiao)||"amazonas".equals(regiao)||"rondonia".equals(regiao)||"tocantins".equals(regiao)){
            System.out.println("Regiao Norte");
	}
	else{
            System.out.println("Outra regiao");
	}
    }
}

File: Feitos em C++/2803.cpp
#include<bits/stdc++.h>
using namespace std;
int main(){
    string regiao;
    cin>>regiao;
    if(regiao=="para"||regiao=="roraima"||regiao=="acre"||regiao=="amapa"||regiao=="amazonas"||regiao=="rondonia"||regiao=="tocantins"){
    	cout<<"Regiao Norte"<<endl;
	}
	else{
		cout<<"Outra regiao"<<endl;
	}
	return 0;
}
